content/java/concepts/math-methods/terms/incrementExact/incrementExact.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: '.incrementExact()'
+
+Description: 'Returns the argument incremented by one, throwing an exception if the result overflows the datatype.'
+Subjects:
+- 'Computer Science'
+  Tags:
+- 'Functions'
+- 'Methods'
+- 'Arithmetic'
+
+  CatalogContent:
+- 'learn-java'
```

Comment:
```suggestion
Title: '.incrementExact()'
Description: 'Returns the argument incremented by one, throwing an exception if the result overflows the data type.'
Subjects:
- 'Computer Science'
  Tags:
- 'Functions'
- 'Methods'
- 'Arithmetic'
  CatalogContent:
- 'learn-java'
```

Reply:
Removing these extra lines will make the content overlap. Added screenshot. Please comment
<img width="1228" alt="Screenshot 2022-10-21 at 4 22 55 PM" src="https://user-images.githubusercontent.com/6908935/197327884-049a7bbc-9f5e-40f4-ae70-566fd9ae589d.png">

Reply:
Not sure where you are previewing the Markdown, but when I preview on GitHub, the metadata renders expected w/ the suggested changes:

<img width="1382" alt="Screen Shot 2022-10-24 at 7 41 41 AM" src="https://user-images.githubusercontent.com/23144845/197517896-363d9c89-6af9-4455-b8b9-fdab807aa4ca.png">

Reply:
You can certainly go off an published entry's meta-data to get a sense of what we're looking for: https://github.com/Codecademy/docs/edit/main/content/java/concepts/math-methods/terms/decrementExact/decrementExact.md

Reply:
@Dusch4593 , Thank you for guiding me and sharing the screenshot. I have made the necessary changes. can you please take a look.